### PR TITLE
i18n(zh-cn): translate `distribute/snapcraft.mdx`

### PR DESCRIPTION
**Pull request by GuoJikun**
#### Description

- What does this PR change? Give us a brief description. 
- Closes #

### CODE REVIEW

**Comment by GuoJikun** [comment]
@vasfvitor Does this need to be changed?

image

File: src/content/docs/zh-cn/distribute/snapcraft.mdx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+      cp -r here/* .
+    organize:
+      usr/bin/app: bin/app
+```
+
+### 解释
+
+- `name` 变量定义了你的应用程序的名称，并且必须设置为之前注册的名称。
+- `base` 变量定义了你正在使用的核心。
+- `version` 变量`定义了版本，并且应该随着源代码库的每次更改而更新。
```
Comment (dklassic) [inline src/content/docs/zh-cn/distribute/snapcraft.mdx]:
```suggestion
- `version` 变量定义了版本，并且应该随着源代码库的每次更改而更新。
```
Just this small mistyped symbol

**Review by dklassic — CHANGES_REQUESTED** [review]

**Review by dklassic — APPROVED** [review]

**Comment by GuoJikun** [comment]
@dklassic https://github.com/tauri-apps/tauri-docs/actions/runs/10032636847
image

**Comment by dklassic** [comment]
> @dklassic https://github.com/tauri-apps/tauri-docs/actions/runs/10032636847 image

I think the workflow is just busted, we’re good.

**Pull request merged**